src/olympia/users/urls.py

```diff
@@ -41,19 +45,22 @@
         RedirectView.as_view(pattern_name='users.login', permanent=True),
         name='users.register'),
     url('^migrate', views.migrate, name='users.migrate'),
-    url(r'^pwreset/?$', anonymous_csrf(auth_views.password_reset),
+    url(r'^pwreset/?$',
+        migration_on(anonymous_csrf(auth_views.password_reset)),
```

Comment:
This will call `waffle_switch()` directly in `urls.py`, meaning we won't be able to toggle the switch without a server restart. In addition, this might cause database queries a little too early, at import time...